controllers/MessagesController.ts

```diff
@@ -44,7 +44,28 @@ export class MessagesCtrl {
             `)
         }
 
-    } 
+    }
+    
+    public async sendOneToMany(req: Request, res:Response ) : Promise<any>{

```

Comment:
the order of the structure must be immutable since it must be completely identical together with the emmit